app/controllers/events.controller.js

```diff
@@ -0,0 +1,165 @@
+const Event = require('../models/event');
+
+module.exports = {
+showEvents: showEvents,
+showSingle: showSingle,
+seedEvents: seedEvents,
+showCreate: showCreate,
+processCreate: processCreate,
+showEdit : showEdit,
+processEdit : processEdit,
+deleteEvent : deleteEvent,
+raisePriority : raisePriority,
+lowerPriority:lowerPriority
+}
+
+
+//show all events
```

Comment:
Is this comment really necessary? I think that the code is speaking for itself, unless you plan to use this as a tutorial in the future?